### ISSUE TITLE
simplifying mesh crashes (at the very end of the process)

### ISSUE THREAD

**Issue by PLeoncini** [cnr-isti-vclab/meshlab#311]
Import the obj model in https://www.dropbox.com/sh/5qgg1bslhpomdk0/AACaAq-NImStnMe6_ncKVDkLa?dl=0 (origin: https://nasa3d.arc.nasa.gov/detail/nmss-sev)
then Filters -> Remeshing, Simplification and Reconstruction -> Simplification: Quadratic Edge Collapse Decimation (with texture)
in the panel: Percentage reduction: 0.2 (the rest at default settings)
then Apply - the process almost arrives at the end, yet instead of showing the simplified mesh the program crashes.

Thank you very much,

Paolo Leoncini
CIRA Centro Italiano Ricerche Aerospaziali
Virtual Reality Lab

**Comment by mcallieri** [comment]
Hi...
various things:
- texture mapping is inconsistent: there are some faces without an UV parametrization. This is a problem for the current version (but we should have fixed it)
- the crash is due to a stupid thing. The simplification succeed, but the software crashes as it tries to display quads that have been destroyed by the simplification. if, BEFORE starting the simplification, you disable wireframe rendering and "polygon mode" in the rendering mode panel. It is a known bug, we are trying to fix it,

Anyway, try using the new beta version, but still remember to disable the wireframe and polygon mode before simplify. It should work.
The current WIN64 build can ba found here

https://ci.appveyor.com/project/cignoni/meshlab/build/artifacts

It is not an installer, just a zipped folder with a "portable" executable. We are starting testing, now, while fixing few more issues. We have used on some machines, here, and seems working. I have no idea, however if it has ALL the possible dll and Runtime that may be needed.

If there is an ongoing build, the site will write "The build job does not contain any artifacts.". In that case, you may wait for the build to end or just click "HISTORY", select the last succesful build, and then "ARTIFACTS"

Finally, I would say that this is a hand-modeled 3D model with many sub-parts. MeshLab is not the best tool to manipulate highly structured models like this one...

**Comment by jonolave** [comment]
Any update on this? I have the same problem on Mac. But even though I turn off both the
wireframe and polygon modes Meshlab crashes.

**Comment by PLeoncini** [comment]
IIRC I eventually solved with the following option to meshlabserver: -m vn
wt (instead of vt - don't ask me why).

Il giorno ven 1 mar 2019 alle ore 12:30 Jon Olav Eikenes <
notifications@github.com> ha scritto:

**Comment by PLeoncini** [comment]
Sorry, my reply seems quite off-topic - my very first problem was a similar
problem yet in meshlabserver, which was solved as I explained.

Il giorno ven 1 mar 2019 alle ore 12:50 Paolo Leoncini <leoncinip@gmail.com>
ha scritto:

**Comment by briana-jin-zhang** [comment]
@mcallieri The link doesn't work any more.

**Comment by mcallieri** [comment]
of course it is not working, it was a link to a build made almost three years ago.
MeshLab has been updated regularly, and there are new releases every month.

**Comment by briana-jin-zhang** [comment]
It works on my mac though, which is strange.

**Comment by alemuntoni** [comment]
It seems to me that the last build does not crash with the shared input model. 
If you still experience this issue, please open another issue.
